xwiki-platform-distribution/xwiki-platform-distribution-war-dependencies/pom.xml

```diff
@@ -300,7 +300,7 @@
     </dependency>
     <dependency>
       <groupId>org.xwiki.rendering</groupId>
-      <artifactId>xwiki-rendering-macro-message</artifactId>
+      <artifactId>xwiki-platform-rendering-macro-message</artifactId>
       <version>${rendering.version}</version>
```

Comment:
Wrong version.

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/commit/5b087f8307cb42f4879090e830147089a3af9f22 :+1: Thank you :)